code/modules/admin/db_ban/functions.dm

```diff
@@ -53,9 +53,9 @@
 			blockselfban = 1
 			kickbannedckey = 1
 
-	if( !bantype_pass ) return
-	if( !istext(reason) ) return
-	if( !isnum(duration) ) return
+	if(!bantype_pass) return
+	if(!istext(reason)) return
+	if(!isnum(duration)) return
```

Comment:
Up to you, but this is against standards

Reply:
out of scope